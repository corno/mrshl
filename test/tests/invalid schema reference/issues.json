[
    [ "deserialization", "error", "'/dev/schemas/FIXME' not found", 1, 3, 1, 8 ],
    [ "deserialization", "warning", "ignoring invalid internal schema", 1, 3, 1, 8 ]
]