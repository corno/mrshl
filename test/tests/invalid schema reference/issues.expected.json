[
	[
		"deserialization",
		"error",
		"schema not found",
		1,
		3,
		1,
		8
	],
	[
		"deserialization",
		"warning",
		"ignoring invalid schema reference",
		1,
		3,
		1,
		8
	]
]